plugins/labhub.py

```diff
@@ -387,3 +388,65 @@ def pr_stats(self, msg, match):
                     state=type(self).community_state(pr_count)
                  )
         yield reply
+
+    @re_botcmd(pattern=r'pr\s+list\s+([^/]+)',  # Ignore LineLengthBear, PyCodeStyleBear
+               re_cmd_name_help='pr list <repo>',
+               flags=re.IGNORECASE)
+    def pr_list(self, msg, match):
+        """List PRs ready to be merged."""  # Ignore QuotesBear
+        repo_name = match.groups(1)[0]
+
+        try:
+            mrs = self.REPOS[repo_name].merge_requests()
+        except KeyError:
+            return 'Repository doesn\'t exist.'
```

Comment:
use double quotes to contain a single quote.
(or is our QuoteBear causing problems again?)

Reply:
Nope, everything is alright, fixed it